tests/components/backup/test_sensors.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for the sensors of the Backup integration."""
+
+from unittest.mock import patch
+
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.backup.const import DOMAIN
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from .common import setup_backup_integration
+
+from tests.common import snapshot_platform
+
+
+async def test_setup_sensors(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test setup backup config entry."""
+    with patch("homeassistant.components.backup.PLATFORMS", [Platform.SENSOR]):
+        await setup_backup_integration(hass, with_hassio=False)
+        await hass.async_block_till_done(wait_background_tasks=True)
+
+    entry = hass.config_entries.async_entries(DOMAIN)[0]
+    await snapshot_platform(hass, entity_registry, snapshot, entry.entry_id)
```

Comment:
We should test updating state. This only checks that the sensors are created in the state machine with idle/unknown state.

Reply:
I got it work to test that the manager state is changing during backup generation, but the timestamps doesn't get updated and I've no idea how to do this - all existing tests I could find, just manipulate the backup storage data and check if the websocket-call returns the data as expected

Reply:
This test should work to follow:

https://github.com/home-assistant/core/blob/1c1a950c05db4b2b506cff6b00b7cefce2c2e8df/tests/components/backup/test_websocket.py#L489-L648